### ISSUE TITLE
Set up shipping task: `Rest of the world` should be clickable

### ISSUE THREAD

**Issue by juliaamosova** [woocommerce/woocommerce-admin#4269]
**Is your feature request related to a problem? Please describe.**

Filing this on behalf of @Aljullu. Reference: pb22l9-8V-p2#comment-398 but I also agree with this statement.

Set up shipping step: ideally, the text `Rest of the world` would be a label so clicking on it toggles the checkbox:

![rest_of_the_world](https://user-images.githubusercontent.com/19143190/80816040-ca18bf80-8b9c-11ea-8c8b-ca7ea9477281.gif)

**Describe the solution you'd like**

`Rest of the world` should be a label so clicking on it toggles the checkbox. 

**Describe alternatives you've considered**

N/A. 

**Should this be prioritized? Why?**

Not a super high priority but will make it for nice user experience. 

**Additional context**

N/A.
